packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -391,9 +391,10 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
         //   2. The amount of gas provided to the call to the target contract is at least the gas
         //      limit specified by the user. If there is not enough gas in the callframe to
         //      accomplish this, `callWithMinGas` will revert.
-        // Additionally, if there is not enough gas remaining to complete the execution after the
-        // call returns, this function will revert.
-        bool success = SafeCall.callWithMinGas(_tx.target, _tx.gasLimit, _tx.value, _tx.data);
+        // Additionally, we use 0 `_reservedGas` here because it does not matter if this context
+        // reverts with "Out of Gas." This is because the user will be able to re-attempt
+        // finalization of the withdrawal with a higher gas limit.
+        bool success = SafeCall.callWithMinGas(_tx.target, _tx.gasLimit, _tx.value, 0, _tx.data);
```

Comment:
nit: prefer kwargs syntax when its unclear what the arguments have become, ie the magic `0`